### PR TITLE
Added strings to the french translation

### PR DESCRIPTION
**Pull request by nazmifr**
Added missing strings, IDK if it's that the good way to add strings and if the text will fit nicely in the interface. Please tell me how if it's not the proper way and want me to contribute.

### CODE REVIEW

File: src/locales/fr/translation.json [hunk 1]
```diff
@@ -1,4 +1,19 @@
 {
+    "Sort notes by": "Trier les notes",
+    "Theme": "Thème",
+    "Notebooks in drawer": "Bloc-notes dans le menu",
+    "Spaces per indent": "Espaces pour indenter",
+    "Text editor": "Éditeur de texte",
+    "Your key": "Votre clé",
+    "Change passphrase": "Changer la phrase de passe",
```
Comment (funilrys) [inline src/locales/fr/translation.json]: "passphrase" == "phrase secr`ete" or "mot de passe" (I do not have the accent on this but you got it :wink: )

File: src/locales/fr/translation.json [hunk 2]
```diff
@@ -1,4 +1,19 @@
...
+    "Synchronization method": "Service de synchronisation",
```
Comment (funilrys) [inline src/locales/fr/translation.json]: M´ethode de synchronisation

File: src/locales/fr/translation.json [hunk 3]
```diff
@@ -1,4 +1,19 @@
...
+    "Erasure options": "Option d'effacement",
+    "Wipe": "Supprimer",
+    "Erase user profile on this computer": "Suprimer le profil d'utilisateur sur cet ordinateur",
```
Comment (funilrys) [inline src/locales/fr/translation.json]: de cet ordinateur

File: src/locales/fr/translation.json [hunk 4]
```diff
@@ -1,4 +1,19 @@
...
+    "Backup your private key": "Sauvegarder votre clé privée",
+    "Backup": "Sauvegarder",
+    "Backup everything": "Tout sauvegarder",
+    "Recover from backup": "Restaurer une sauvegarde",
```
Comment (funilrys) [inline src/locales/fr/translation.json]: Maybe: Restaurer depuis une sauvegarde

**Comment by daed** [comment]
@nazmifr That's exactly what we need.  I'm not sure the best way to confirm text fit.  In this case it looks okay.  I wonder if there is some tool that we can use to validate that beyond manual confirmation.

I'll put that in the testing to-do.

**Comment by funilrys** [comment]
Hi @daed we could use https://docs.transifex.com/projects/open-source-project :smile_cat: I'm a reviewer in some big or small project there and I do think that it's great to use :+1:

**Pull request merged**